app/helpers/async_stylesheets_helper.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module AsyncStylesheetsHelper
```

Comment:
Stole the implementation here from https://github.com/discourse/discourse/blob/0c13c91f848e60be5e321f57a057f12809e8fc3f/app/helpers/splash_screen_helper.rb